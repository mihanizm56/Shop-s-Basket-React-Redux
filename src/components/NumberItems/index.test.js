import React from "react"
import { shallow } from "enzyme"
import { NumberOfItems } from "."

describe("NumberOfItems", () => {
  it("Renders correct if numberOfItems more than 4", () => {
    const component = shallow(<NumberOfItems numberOfItems={5} />)
    expect(component).toMatchSnapshot()
  })

  it("Renders correct if numberOfItems less than 4", () => {
    const component = shallow(<NumberOfItems numberOfItems={3} />)
    expect(component).toMatchSnapshot()
  })
})
