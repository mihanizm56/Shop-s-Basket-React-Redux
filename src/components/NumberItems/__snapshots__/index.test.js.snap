// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NumberOfItems Renders correct if numberOfItems less than 4 1`] = `
<styled.p
  numberOfItems={0}
>
  В корзине: 
  <b>
    3 товара
  </b>
</styled.p>
`;

exports[`NumberOfItems Renders correct if numberOfItems more than 4 1`] = `
<styled.p
  numberOfItems={0}
>
  В корзине: 
  <b>
    5 товаров
  </b>
</styled.p>
`;
